airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/development/MigrationDevCenter.java

```diff
@@ -106,6 +111,12 @@ private void dumpSchema() {
     }
   }
 
+  private static void initializeDatabase(final PostgreSQLContainer container) {
+    final var containerDelegate = new JdbcDatabaseDelegate(container, "");
+    ScriptUtils.runInitScript(containerDelegate, "configs_database/schema.sql");
+    ScriptUtils.runInitScript(containerDelegate, "jobs_database/schema.sql");
```

Comment:
@jdpgrailsdev, I think this is incorrect. Each migration dev center should run specific its own init script, i.e. the config migration dev center should only run the configs initial script.

Currently when I run schema dump for one database, it will include the schemas for both configs and jobs database.

The schema dump is still correct in production because it is updated by `ConfigsDatabaseMigratorTest` and `JobsDatabaseMigratorTest`, which only does not have this behavior.